### PR TITLE
Fixes temp autoreport, and other improves

### PR DESCRIPTION
**Pull request by chepo92**
added some comments
temp autoreport now works - this had to be fixed bc M109 triggers autoreport in the machine 
 echo:busy , echo: cold print and error are detected
for a M109 (long running command) has more timeout
added some telnet messages

### CODE REVIEW

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino [hunk 1]
```diff
@@ -765,50 +771,39 @@ void ReceiveResponses() {
...
-          serialResponse = "";
-
+          GOT_VALID_RESPONSE();
           unsigned int cmdLen = commandQueue.popAcknowledge().length();  // Command has been processed by printer, buffer has been freed
           printerUsedBuffer = max(printerUsedBuffer - cmdLen, 0u);
           resetSerialReceiveTimeout();
-          if (lastCommandSent.startsWith("M155 S") && !lastCommandSent.startsWith("M155 S0")) {
-            fwAutoreportTempCapEn= true ;
-          }
+          if (lastCommandSent.startsWith("M155 S1"))
```
Comment (chepo92) [inline ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: Don't be so specific, imagine If a gcode contains M155 S0, fwAutoreportTempCapEn should be false and don't expect a line with autoreportemps, also what about M155 S2? I used S10 above to not flood telnet with temp responses while debuging, but it should report each 2-5 secs

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino [hunk 2]
```diff
@@ -765,50 +771,39 @@ void ReceiveResponses() {
   while (Serial.available()) {
     char ch = (char)Serial.read();
     serialResponse += ch;
-    serialReceiveTimeoutTimer += 500;   // Once a char is received timeout may be shorter
+    serialReceiveTimeoutTimer = millis() + 500;   // Once a char is received timeout may be shorter
```
Comment (chepo92) [inline ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: I will have to test this again
(Initialy doing = millis() + 500; was messing with long running commands (line 760 761), equaling to millis() is like a restart (loosing the 10000msec that line 761 gave), but you don't want a restart each time you recieve a char, just give it more time to be completed, eg M109 SXX is a "long running command (there are more)" that doesn't reply "ok" until the heating is done. But now we have echo:busy detection  it adds timeout so maybe it works)

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino [hunk 3]
```diff
@@ -24,7 +24,7 @@ DNSServer dns;
 // Configurable parameters
 #define SKETCH_VERSION "2.0"
 #define PRINTER_RX_BUFFER_SIZE 0        // This is printer firmware 'RX_BUFFER_SIZE'. If such parameter is unknown please use 0
-#define TEMPERATURE_REPORT_INTERVAL 2   // Ask the printer for its temperatures status every 2 seconds
+#define TEMPERATURE_REPORT_INTERVAL 10  // Ask the printer for its temperatures status every 10 seconds
```
Comment (chepo92) [inline ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: Forgot to put it back to 2 sec, just did't wanted so many messages in telnet.

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino (hunk 2 above)
Comment (GMagician) [reply ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: Yes but this is reset every byte is received...once a frame is coming I think it shouldn't need a lot of time  be received...

Comment (GMagician) [reply ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: Again long command should give you a "busy" and busy comes every 2 sec.. (M109 will need time but busy comes quickly

Comment (GMagician) [reply ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: I even think that 500ms is a lot of time for char to char....

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino (hunk 3 above)
Comment (GMagician) [reply ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: Ok I'll fix

File: ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino (hunk 1 above)
Comment (GMagician) [reply ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino]: I already fixed it, thanks

**Pull request merged**